data_architect_misc/data_transformer/transform_functions/common_transform_functions.py

```diff
@@ -343,7 +343,7 @@ def drop_rows_with_empty_values_or_nan_values(
             Dataframe with rows dropped (if matches were found).
         """
         for i, col_name in enumerate(list_of_col_names):
-            df = df[~((df[col_name] == "") | (df[col_name].isnull()))]
+            df = df.loc[~((df[col_name].astype(str) == "") | (df[col_name].isnull()))]
```

Comment:
Do we need to convert the column to string? I see that you are using this method to drop rows in an integer/float column, "TOTAL DOLS (000)". Shouldn't `df[col_name].isnull()` drop the null/NaN values in the integer/float column? If not, is that because these float values are read as pandas.object?

I ask this because the conversion of data to string costs computing time and if the user is using this method (think of converting 100,000 rows to string first before checking them for empty string). If the `isnull()' check is not working for integers/floats, we need to find out why and update that part instead of forcing to convert integers into string for empty string check.